package.json

```diff
@@ -14,29 +14,32 @@
     "vue": "^3.0.0-0"
```

Comment:
Il y a une vraie version 3.0.X

Reply:
J'ai upgrade en 3.0.1 (latest stable), ça fonctionne plus malgré 0 erreur haha

Reply:
Upgraded to 3.0.1.